exercises/APE1/0103

```diff
@@ -0,0 +1,114 @@
+For this exercise, let us forget the names of the pirates; instead, let us denote pirates with a number, coinciding with their position in the hierarchical order.
+Hence, we have the following names
+\begin{enumerate}
+\item Cabin Boy, pirate $p_{1}$
+\item Mate, pirate $p_{2}$
+\item Navigator, pirate $p_{3}$
+\item Ship Master, pirate $p_{4}$
+\item Captain, pirate $p_{5}$
+\end{enumerate}
+
+Of course, if we imagine a crew with more than five pirates, we can use the same idea to denote them: if $i < j$, then pirate $p_j$ is higher in the hierarchical order than pirate $p_i$.
+
+
+Let us analyse each horizontal block. As a reminder, when the crew has $n$ pirates, the most highly ranked pirate proposes a particular division of the treasure. This pirate will always be pirate $p_{n}$, and he will need strictly more than $\floor*{\dfrac{n}{2}}$ votes to stay alive; in other words, he will need at least $\ceil*{\dfrac{n+1}{2}}$ votes to stay alive. However, since he will always vote for his proposition, we will need at least $\ceil*{\dfrac{n-1}{2}}$ other votes to stay alive.
```

Comment:
I have added two DeclarePairedDelimiters in the file linma2345.sty. Didn't it work?
I copy-paste them here
\DeclarePairedDelimiter{\floor}{\lfloor}{\rfloor}
\DeclarePairedDelimiter{\ceil}{\lceil}{\rceil}